src/index.html

```diff
@@ -11,8 +12,61 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="linthtml"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__content">
+        <div class="card__image"></div>
+
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="card__product-code">Product code: 195434</div>
+
+        <div class="card__rate">
+          <div class="card__rate-stars">
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
+            <div class="card__rate-stars-star"></div>
```

Comment:
it should be a separate block

and move that stars styles to separate scss file